Dockerfile

```diff
@@ -17,7 +17,8 @@ RUN crudini --set /etc/ironic-inspector/inspector.conf DEFAULT auth_strategy noa
     crudini --set /etc/ironic-inspector/inspector.conf processing always_store_ramdisk_logs true && \
     crudini --set /etc/ironic-inspector/inspector.conf pxe_filter driver noop && \
     crudini --set /etc/ironic-inspector/inspector.conf processing node_not_found_hook enroll && \
-    crudini --set /etc/ironic-inspector/inspector.conf discovery enroll_node_driver ipmi
+    crudini --set /etc/ironic-inspector/inspector.conf discovery enroll_node_driver ipmi \
```

Comment:
I think you're missing "&&" at the end of the line

Reply:
Good point. And building images locally doesn't work for me :(